Assets/Scripts/Units/Melee Unit/AxeUnitChaseState.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace RogueApeStudio.Crusader.Units.MeleeUnit
+{
+    public class AxeUnitChaseState : IAxeUnitState
+    {
+        public void EnterState(AxeUnit AxeUnit)
```

Comment:
From a style point of view, make it an empty one liner.
Also, the arguments do not conform to the code convetion. They should be camelCase.

```csharp
public void EnterState(AxeUnit axeUnit) { }
```